docs/meshcloud.project-metering.md

```diff
@@ -68,17 +68,18 @@ statements do as well show charges with a booking and valuta date.
 
 #### Tags and Payment Methods
 
-Chargeback statements capture the billing information associated with the chargeback account at the time
-the chargeback statement is generated. This implies that the payment method that is active on the project at the
-point in time when the chargeback statement is generated will be charged for all line items in the chargeback statement.
+Chargeback statements capture the billing information associated with the chargeback account at the end of the report period
+of its line items. This implies that the payment method that is active on the project at the
+end of the report period of a line item will be set for the according line item.
```

Comment:
This sentence makes sense to explain the workings but it is still a bit hard to understand. I think adding an example will fix that.